lib/telephone_number/version.rb

```diff
@@ -1,3 +1,3 @@
 module TelephoneNumber
-  VERSION = "1.1.1"
+  VERSION = "1.2.0.pre"
```

Comment:
Would this be better categorized as a [release candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate)?

> A release candidate (RC) is a beta version with potential to be a final product, which is ready to release unless significant bugs emerge. In this stage of product stabilization, all product features have been designed, coded and tested through one or more beta cycles with no known showstopper-class bugs.